DuckDuckGo/AdAttribution/AdAttributionPixelReporter.swift

```diff
@@ -33,17 +33,9 @@ final actor AdAttributionPixelReporter {
     private let pixelFiring: PixelFiringAsync.Type
     private var isSendingAttribution: Bool = false
 
-    private let inconsistencyMonitoring: AdAttributionReporterInconsistencyMonitoring
-    private let attributionReportSuccessfulFileMarker = BoolFileMarker(name: .isAttrbutionReportSuccessful)
-
     private var shouldReport: Bool {
         get async {
-            if let attributionReportSuccessfulFileMarker {
-                // If marker is present then report only if data consistent
-                return await !fetcherStorage.wasAttributionReportSuccessful && !attributionReportSuccessfulFileMarker.isPresent
-            } else {
-                return await fetcherStorage.wasAttributionReportSuccessful
-            }
+            return await !fetcherStorage.wasAttributionReportSuccessful
```

Comment:
There was a mistake in previous PR - there was no negation (https://github.com/duckduckgo/iOS/pull/3510/files#diff-6fd446bd1dcbca6190f5371b19e0abc7568e824facf0aac081c01e84e034fbcfR44), but assuming `attributionReportSuccessfulFileMarker` was present at all times it probably wasn't an issue.